New Extensions book for OLMv1 (#78462)
openshift/openshift-docs: extensions/of-terms.adoc

:_mod-docs-content-type: ASSEMBLY
[id="of-terms"]
= Operator Framework glossary of common terms
include::_attributes/common-attributes.adoc[]
:context: of-terms

toc::[]

> This topic provides a glossary of common terms related to the Operator Framework, including {olmv1-first}.

Reviewer: ```suggestion
The following terms are related to the Operator Framework, including {olmv1-first}.
```
Avoid self-referential wording. (https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences#style)
